app/src/main/java/cm/aptoide/pt/app/view/donations/DonationsAnalytics.java

```diff
@@ -0,0 +1,47 @@
+package cm.aptoide.pt.app.view.donations;
+
+import cm.aptoide.analytics.AnalyticsManager;
+import cm.aptoide.analytics.implementation.navigation.NavigationTracker;
+import java.util.HashMap;
+
+public class DonationsAnalytics {
+
+  public static final String DONATIONS_INTERACT = "Donations_Dialog_Interact";
+  public static final String ACTION = "Action";
+  public static final String PACKAGE_NAME = "Package_Name";
```

Comment:
This is not with uper case on the ticket, i dont know if it is problematic or not